### PR TITLE
fix #32, allow ogv video uploads

### LINKED ISSUE #32: Wordpress doesn't accept OGV
Client got Wordpress error when uploading their OGV. 

>Sorry this filetype is not permitted for security reasons. 

verify that WP accepts OGV. fix it or remove this filetype option.

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

**Comment by mikeyrayvon** [comment]
lol did you test this?

**Comment by mikeyrayvon** [comment]
cause shit aint work

**Comment by mixcas** [comment]
Lol Unlike u, I always test my code

**Comment by varffs** [comment]
I don't always test my code, but when I do test my code it is the most interesting code in the world

File: lib/functions-filters.php [hunk 1]
```diff
@@ -45,3 +45,12 @@ function add_lazysize_on_srcset($attr) {
 
 }
 add_filter('wp_get_attachment_image_attributes', 'add_lazysize_on_srcset');
+
+// Allow for OGV file type uploads
+add_filter( 'upload_mimes', 'allowed_filetypes', 1, 1 );
```
Comment (varffs) [inline lib/functions-filters.php:50]: this code is correct but doesnt work for me. Why the `1, 1` extra vars?

**Comment by mixcas** [comment]
u guys <3

Idk, its night here but i can check it tomorrow. im 99% sure it's "working on my local"

File: lib/functions-filters.php (hunk 1 above)
Comment (mixcas) [reply lib/functions-filters.php:50]: > $priority
(int) (Optional) Used to specify the order in which the functions associated with a particular action are executed. Lower numbers correspond with earlier execution, and functions with the same priority are executed in the order in which they were added to the action.
Default value: 10
$accepted_args
(int) (Optional) The number of arguments the function accepts.
Default value: 1

**Comment by mixcas** [comment]
It's working on my local perfectly. Tried with diff files encoded with different apps (although both apps use ffmpeg)

You need to debug this on your own

**Comment by varffs** [comment]
I think we should just scrap this. It isn't needed and debugging will take longer than it is worth. But we should remove the .ovg options from the codebase entirely

**Comment by mikeyrayvon** [comment]
ok i'll do that now

**Comment by mikeyrayvon** [comment]
#47 should close this PR

**Pull request closed without merging**